api/db/migrate/20190516205625_add_columns_to_assignment.rb

```diff
@@ -0,0 +1,10 @@
+class AddColumnsToAssignment < ActiveRecord::Migration[5.2]
+  def change
+    add_column :assignments, :hours, :integer
+    add_column :assignments, :pay1, :float
+    add_column :assignments, :pay2, :float
```

Comment:
It's unclear to me what is meant by "pay1" and "pay2". Can this be clarified with a comment or change of names ?

Reply:
Actually, I plan to remove these attributes. Even though it was discussed during the meeting on Thursday that we should use them, there's apparently a plan to move them to a new table called `fund_source`. I'm kind of waiting for a final decision in case there's anymore changes...

Reply:
I see. Is there any harm in removing them now since they'll be find-able under version control anyways ? I would be tempted to remove them for now and keep a note somewhere when fund_source gets created. We can always put them back.